go/stats/timings.go

```diff
@@ -180,7 +180,18 @@ func (t *Timings) Label() string {
 	return t.label
 }
 
-var bucketCutoffs = []int64{5e5, 1e6, 5e6, 1e7, 5e7, 1e8, 5e8, 1e9, 5e9, 1e10}
+var bucketCutoffs = []int64{
```

Comment:
This isn't needed. Can revert although I kind of like it.

Reply:
I guess these aren't strictly seconds...it depends on the unit of the metric. Will revert.